doc/source/whatsnew/v1.1.0.rst

```diff
@@ -109,7 +109,7 @@ Datetimelike
 Timedelta
 ^^^^^^^^^
 
--
+- Bug in constructing a :class:`Timedelta` with a high precision integer that would round the :class:`Timedelta` components (:issue:`31354`)
```

Comment:
BTW, I'm not sure if it was already covered by the earlier PR, but it seems like the change in precision to `total_seconds()` from "dubiously nanosecond-precision" to "microsecond-precision" should probably show up in the release notes. It seems like this is the only entry under `Timedelta`, so maybe it was done silently? Or was the other change already released?